VideoLocker/src/main/java/org/edx/mobile/services/ServiceManager.java

```diff
@@ -73,11 +73,13 @@ public String getUrl() {
                     String username = URLEncoder.encode(pref.getCurrentUserProfile().username, "UTF-8");
                     String block_counts = URLEncoder.encode("video", "UTF-8");
                     String requested_fields = URLEncoder.encode("graded,format,student_view_multi_device", "UTF-8");
-                    String student_view_data = URLEncoder.encode("video", "UTF-8");
+                    String student_view_data = URLEncoder.encode("discussion,video", "UTF-8");
```

Comment:
nit: RestApiManager.java has these the in the opposite order. Should we make these the same order for consistency? "video,discussion"